ts/Stock/StockToolsBindings.ts

```diff
@@ -348,9 +354,12 @@ bindingsUtils.manageIndicators = function (
             series.remove(false);
 
             if (indicatorsWithAxes.indexOf(series.type) >= 0) {
-                const removedYAxisHeight = yAxis.options.height;
+                const removedYAxisProps = {
+                    height: yAxis.options.height,
+                    top: yAxis.options.top
+                } as Highcharts.AxisPositionsObject;
```

Comment:
There is something wrong with `AxisPositionsObject` interface declaration if you still need to cast it here like this.

Reply:
Additionally, I think there is no need for adding the `Object` word into that interface's name.

Reply:
As you can see below, the `removedYAxisHeight` was previously casted as well (to string).  Since we can only remove the axis we previously created by GUI, these options are certainly strings, so I decided to cast them here, since this is not an issue.